### DISCUSSION TITLE
Dynamically manage OIDC client

### DISCUSSION THREAD

**Discussion by ii64** [authelia/authelia#4031] [Q&A] [answered]
Hello, is Authelia IDP OIDC supports dynamic/runtime management clients? I am currently hard coded all the clients on the config file as explained in https://www.authelia.com/configuration/identity-providers/open-id-connect/, but then I need to restart the Authelia process which sounds inconvenient. Or maybe any other possible alternatives?
Thanks

**Comment by james-d-elliott** [comment, marked as answer]
We will add this eventually via DCRP but it's not currently supported.
